src/style.css

```
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars_star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars_star:nth-child(-n + 1) {
```

```suggestion
.stars--1 .stars_star:nth-child(-n + 1),
.stars--2 .stars_star:nth-child(-n + 2),
... {
```
Consider using commas to group selectors